pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: lints
+description: Lints that represent the Dart style guide
```

Comment:
Maybe we should generalize the mission statement?

> Lints that enforce recommended Dart style

> Official Dart lint rules

Do we want to include flutter as well?

> Official Dart and Flutter lint rules

Dunno...

(Happy to land though and iterate.)

Reply:
+1 to keeping this a little more general.

Reply:
Done